engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -313,7 +313,7 @@ public void map(BatchSource<InputRow> source, BatchSink<Collection<Object[]>> si
         } catch (RuntimeException e) {
             boolean alreadyFailed = failed.getAndSet(true);
             if (!alreadyFailed) {
-                throw e;
+                throw new RuntimeException("Remote transformer failed: " + e.getMessage(), e);
```

Comment:
This is intentional. Previously, the REST Client error message contained something like "Remote transformer failed: ....". But the client can be used in another situations that from the transformer. So we removed the "Remote transformer" part from the error message in REST client. But we want that message "prefix" when doing a call in the RemoteTransformer.
So the new RuntimeException here is used to enhance the error message.